build-patched-kao-kcmo-images.sh

```diff
@@ -38,14 +38,14 @@ function check_pull_secret() {
 check_pull_secret
 
 HOST_ARCH=$(uname -m)
-MIRROR=${MIRROR:-https://mirror.openshift.com/pub/openshift-v4/$HOST_ARCH/clients/ocp}
+MIRROR=${MIRROR:-https://mirror.openshift.com/pub/openshift-v4/$HOST_ARCH/clients/ocp-dev-preview}
```

Comment:
A follow-up commit will be needed to change tihs to `ocp` rather than `ocp-dev-preview`, and to change `candidate` to `latest` in this script and in snc.sh.
However, there is no `clients/ocp/{latest,candidate,stable}-4.14` yet, which would indicate 4.14 is unreleased for now? 
Maybe it's best to keep `master` to track released openshift versions, and only merge 4.14 after we merged a patch to use `clients/ocp` and `latest-4.14`?
In the mean time, we can merge `microshift.sh: switch to release-4.14 ` in the 4.14 branch.

Reply:
https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/candidate-4.14/ is available and yes we need a follow up commit for that or may be this should be part of https://github.com/crc-org/snc/commit/47f36c734f716bc8634de9e77affd2ce7345ee6d where I am updating the version info.

Reply:
> https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/candidate-4.14/ is available 

But we need `latest` for our builds?

Reply:
latest is also available now https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest-4.14/